src/main/java/org/radarbase/output/source/SourceStorageFactory.kt

```diff
@@ -23,9 +25,11 @@ class SourceStorageFactory(private val resourceConfig: ResourceConfig, private v
             S3SourceStorage(minioClient, s3Config, tempPath)
         }
         ResourceType.HDFS -> {
```

Comment:
is this still supported?

Reply:
It's only supported if someone adds the HDFS storage factory as an extension JAR. This could be a single project where HDFS is absolutely needed.

Reply:
ah right! Are there any instructions for that in the docs?